src/Modules/PocketWaifu.cs

```diff
@@ -2493,7 +2493,7 @@ public async Task SearchCharacterCardsFromFavListAsync([Summary("czy pokazać ul
         [Alias("which")]
         [Summary("pozwala wyszukać użytkowników posiadających karty z danego tytułu")]
         [Remarks("1 tak"), RequireWaifuCommandChannel]
-        public async Task SearchCharacterCardsFromTitleAsync([Summary("id serii na shinden")] ulong id, [Summary("czy zamienić oznaczenia na nicki?")] bool showNames = false)
+        public async Task SearchCharacterCardsFromTitleAsync([Summary("id serii na shinden")] ulong id, [Summary("czy zamienić oznaczenia na nicki? (opcjonalne)")] bool showNames = false)
```

Comment:
"opcjonalne" is unnecessary

Reply:
Only this or that two in #2457 too, because it looks like it was missed